src/SearchParameters/index.js

```diff
@@ -333,12 +342,68 @@ function SearchParameters(newParameters) {
 
   forOwn(params, function checkForUnknownParameter(paramValue, paramName) {
     if (!this.hasOwnProperty(paramName)) {
-      console.error('Unsupported SearchParameter: `' + paramName + '` (this will throw in the next version)');
+      /*eslint-disable*/
```

Comment:
Should not be needed since console and line length are warnings in our eslint configuration, maybe you want to avoid warnings somehow?